plugins/processors/converter/converter.go

```diff
@@ -452,6 +478,24 @@ func toString(v interface{}) (string, bool) {
 	return "", false
 }
 
+func parseHexadecimal(value string) (float64, error) {
+	i := new(big.Int)
+
+	_, success := i.SetString(value, 0)
+	if !success {
+		return 0, errors.New("unable to parse string to big int")
+	}
+
+	f := new(big.Float).SetInt(i)
+	result, _ := f.Float64()
```

Comment:
I'm not sure returning the big hex number as a float is a good idea.  It will work for hex numbers a little larger than int64, but for bigger hex humbers it will hit the limit of a float64 and start losing precision.  Hex is used when data needs to be bit precise.  Any lossy format will cause trouble for someone.

Reply:
Thanks for reviewing. As I don't believe we can have a metric with a larger number than float64, wouldn't we encounter this problem regardless?

Reply:
I don't think I understood what this PR is trying to do when I commented earlier.  If the user has a very big hex number and sets up a processor to convert it to a float, they should know that floats by nature have precision limitations.  They should expect the conversion to lose precision.

Maybe all we need to do is add a warning to the documentation so all users will be aware that this functionality exists and that it can be a lossy conversion.